src/modules/organizations/containers/organization-container.jsx

```diff
@@ -22,13 +23,25 @@ class OrganizationContainer extends React.Component {
     this.props.dispatch(setCurrentOrganization(null));
   }
 
+  updateOrganization(organizationFields) {
+    organizationFields.name = organizationFields.display_name; // eslint-disable-line no-param-reassign
+    organizationFields.title = organizationFields.name.replace(/\W/g, ''); // eslint-disable-line no-param-reassign
+    apiClient.type('organizations').get(this.props.organization.id)
```

Comment:
that's true. i did it this way because i didn't want to modify the props (React discourages that) and in case someone else modified the organization since the last time you loaded your screen, but we can just use the cached one if that seems to make more sense.